homeassistant/components/tessie/entity.py

```diff
@@ -27,7 +27,9 @@ def __init__(
 
         car_type = coordinator.data["vehicle_config-car_type"]
 
-        self._attr_translation_key = key
+        if not hasattr(self, "_attr_translation_key"):
+            self._attr_translation_key = key
+
```

Comment:
You're also not checking for entity description translation keys

Reply:
Ok ill revert this change. I dont think I need it anyway.